packages/webview_flutter/webview_flutter_wkwebview/lib/src/webview_cupertino_widget.dart

```diff
@@ -0,0 +1,157 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/widgets.dart';
+import 'package:webview_flutter_platform_interface/webview_flutter_platform_interface.dart';
+
+import 'web_kit/web_kit.dart' as web_kit;
+
+/// A [Widget] that displays a [web_kit.WebView].
+class WebViewCupertinoWidget extends StatefulWidget {
+  /// Constructs a [WebViewCupertinoWidget].
+  const WebViewCupertinoWidget({
+    required this.creationParams,
+    required this.callbacksHandler,
+    required this.javascriptChannelRegistry,
+    required this.onBuildWidget,
+    this.configuration,
+    @visibleForTesting this.webViewProxy = const WebViewProxy(),
+  });
+
+  /// The initial parameters used to setup the WebView.
+  final CreationParams creationParams;
+
+  /// The handler of callbacks made made by [web_kit.NavigationDelegate].
+  final WebViewPlatformCallbacksHandler callbacksHandler;
+
+  /// Manager of named JavaScript channels and forwarding incoming messages on the correct channel.
+  final JavascriptChannelRegistry javascriptChannelRegistry;
+
+  /// A collection of properties that you use to initialize a web view.
+  ///
+  /// If null, a default configuration is used.
+  final web_kit.WebViewConfiguration? configuration;
+
+  /// The handler for constructing [web_kit.WebView]s and calling static methods.
+  ///
+  /// This should only be changed for testing purposes.
+  final WebViewProxy webViewProxy;
+
+  /// A callback to build a widget once [web_kit.WebView] has been initialized.
+  final Widget Function(WebViewCupertinoPlatformController controller)
+      onBuildWidget;
+
+  @override
+  State<StatefulWidget> createState() => _WebViewCupertinoWidgetState();
+}
+
+class _WebViewCupertinoWidgetState extends State<WebViewCupertinoWidget> {
+  late final WebViewCupertinoPlatformController controller;
+
+  @override
+  void initState() {
+    super.initState();
+    controller = WebViewCupertinoPlatformController(
+      creationParams: widget.creationParams,
+      callbacksHandler: widget.callbacksHandler,
+      javascriptChannelRegistry: widget.javascriptChannelRegistry,
+      configuration: widget.configuration,
+      webViewProxy: widget.webViewProxy,
+    );
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return widget.onBuildWidget(controller);
+  }
+}
+
+/// An implementation of [WebViewPlatformController] with the WebKit api.
+class WebViewCupertinoPlatformController extends WebViewPlatformController {
```

Comment:
Same name change here.

Reply:
conversation continued in https://github.com/flutter/plugins/pull/4626#discussion_r799729739